src/fides/api/ops/schemas/saas/saas_config.py

```diff
@@ -303,6 +317,10 @@ def get_graph(self) -> Dataset:
                 if param.references:
                     references = []
                     for reference in param.references:
+                        if isinstance(reference, str):
```

Comment:
will do. i'm thinking maybe i should try to refactor some of this reference type-checking and subsequent secret dereferencing into a shared utility method - it may make it feel a bit less sloppy, and also easier to maintain if/when we want to make further updates.

Reply:
I do like the idea of a shared utility method!